src/lib.rs

```diff
@@ -771,10 +771,13 @@ bsize: BlockSize, bo: &BlockOffset) -> f64 {
         return rd_cost;
     }
 
-    let is_sb_on_frame_border = (fi.sb_width - 1) * 16 <= bo.x || (fi.sb_height - 1) * 16 <= bo.y;
+    let bs = mi_size_wide[bsize as usize];
+    let is_too_big =
```

Comment:
SO, 'too_big' means 'partition too big', i.e. current partition being considered straddle on the frame border and "out of bound", correct?